include/pipettes/core/eeprom.hpp

```diff
@@ -0,0 +1,39 @@
+#pragma once
```

Comment:
really like that this file exists and is in pipettes, i think that's a great separation of concerns